src/v3/src/components/ImageWithText/ImageWithText.tsx

```diff
@@ -19,7 +19,7 @@ import { ImageWithTextElement, UISchemaElementComponent } from 'src/types';
 const ImageWithText: UISchemaElementComponent<{
   uischema: ImageWithTextElement
 }> = ({ uischema }) => {
-  const Icon = uischema.options.SVGIcon;
+  const Icon = uischema.options.SVGIcon; // FIXME
```

Comment:
I removed the FIXME because it is working as expected now, but I think when Lester was first working on this, it was taking the SVG file name and trying to use it as a component instead of converting the SVG into a function component. 

However, I confirmed this is working as expected now so no fix needed.